### ISSUE TITLE
result became bad than resnet50

### ISSUE THREAD

**Issue by AItechnology** [zhanghang1989/ResNeSt#37]
I have trained a resnest50-based detection model ( FCOS) using mmdetection framework, but the result on coco dataset became bad about 7%-8%, I  only change the backbone, what happened?

**Comment by Jerryzcn** [comment]
Without see your code, it is hard to tell. There are couple pitfall for swaping resnet with resnest. 1. disable dropblock. 2. make sure batch size is greater than 8 if you are using batch norm

**Comment by AItechnology** [comment]
I just add your resnest module in mmdetection framework and load pretrained resnest50 model, is there anything else I should adjust?

**Comment by zhanghang1989** [comment]
I would suggest wait for our official `mmdetection` implementation. At the meantime, you may use [detectron2-ResNeSt](https://github.com/zhanghang1989/detectron2-ResNeSt).

**Comment by hanjianhua44** [comment]
@zhanghang1989 I want to know how soon that official mmdetection implementation can be released?

I have added code in this repo into mmdetection, but something bother me is that I don't know how to combine using resnest with dcn(deformable conv network), would you give me some idea?

**Comment by zhanghang1989** [comment]
@chongruo has enabled DCN on detectron. @chongruo could you share some experience on that?

**Comment by hanjianhua44** [comment]
@zhanghang1989 I have checked the Dectectron2 fork of ResNeSt, found that the backbone doesn't combine with the dcn.

I simply changed the conv2d of SplAtConv2d into DeformConvPack or ModulatedDeformConvPack. However, for DeformConvPack the loss doesn't convergent, and for ModulatedDeformConvPack, it throws out some errors.

**Comment by zhanghang1989** [comment]
Chongruo is experimenting on this. We tried using DCN with ResNeSt, but the training is not very stable especially for the deeper network (101, 200). 
We found increasing the number of `deform_num_groups` to 2 may be helpful. We can give an update on this when we have a conclusion.

**Comment by zhanghang1989** [comment]
Just in case that you didn't notice, we have released the model using ResNeSt with DCN on detectron2: https://github.com/zhanghang1989/detectron2-ResNeSt

**Comment by hanjianhua44** [comment]
@zhanghang1989 Thanks, I have implemeted DCNv2 successfully on mmdetection, although it can only get 43.2% mAp on mmdetection（conpare to 44.1% on detectron2）.

Hope to see the official implementation!

**Comment by zhanghang1989** [comment]
Thanks for the reply!  I think this issue can be closed for now.
